src/OpenTelemetry/Metrics/MetricAggregators/SumMetricAggregatorLong.cs

```diff
@@ -26,11 +26,15 @@ internal class SumMetricAggregatorLong : IAggregator
         private long sumLong = 0;
         private SumMetricLong sumMetricLong;
         private DateTimeOffset startTimeExclusive;
+        private bool isDeltaValue;
+        private bool isMonotonicValue;
 
-        internal SumMetricAggregatorLong(string name, string description, string unit, Meter meter, DateTimeOffset startTimeExclusive, KeyValuePair<string, object>[] attributes)
+        internal SumMetricAggregatorLong(string name, string description, string unit, Meter meter, DateTimeOffset startTimeExclusive, KeyValuePair<string, object>[] attributes, bool isDeltaValue, bool isMonotonicValue)
         {
             this.startTimeExclusive = startTimeExclusive;
             this.sumMetricLong = new SumMetricLong(name, description, unit, meter, startTimeExclusive, attributes);
+            this.isDeltaValue = isDeltaValue;
+            this.isMonotonicValue = isMonotonicValue;
```

Comment:
this is not required to be part of this PR right? (With UpDownCounter being present, we can make this hardcoded for now, i think)

Reply:
It sets up the prototype. But, the callers all have it hardcoded to true (MapToMetrics).

Reply:
I though .NET counter can take non-monotonic values.